src/azul/chalice.py

```diff
@@ -142,6 +142,20 @@ def _log_response(self, response):
         else:
             log.info('Returning %i response. To log headers and body, set AZUL_DEBUG to 1.', response.status_code)
 
+    def _register_handler(self, handler_type, name, user_handler,
+                          wrapped_handler, kwargs, options=None):
+        """
+        We give all of our handlers a reference to their own names which allows
```

Comment:
This doesn't go into the docstring. It's an implementation detail.

Reply:
"their own names"  is inaccurate. It's the name of the corresponding lambda function.